core/server/api/v2/utils/serializers/output/utils/url.js

```diff
@@ -64,6 +64,21 @@ const forPost = (id, attrs, frame) => {
         ).html();
     }
 
+    // CASE: we always need to return absolute urls for content urls, even in mobiledoc
+    if (attrs.mobiledoc && !localUtils.isContentAPI(frame)) {
+        const imagePathRe = new RegExp(`/${urlService.utils.STATIC_IMAGE_URL_PREFIX}`, 'g');
+
+        const matches = _.uniq(attrs.mobiledoc.match(imagePathRe));
+
+        if (matches) {
+            matches.forEach((match) => {
+                // @NOTE: the utility will skip the match if it's already absolute
+                const absolute = urlService.utils.relativeToAbsolute(match);
+                attrs.mobiledoc = attrs.mobiledoc.replace(new RegExp(match, 'g'), absolute);
```

Comment:
The logic here is very unreliable.
Copied from input serializer. Need to change that before merging.

Reply:
We have to match images and not everything.

- you can also get `<img src="">` in a html card
- [image-1 (/content/images/2019/03/image-1.png)"}]
- [["image",{"src":"/content/images/2019/03/order-1.png"}

Known cases ^

Reply:
Another case:

- gallery cards also have nested images